src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -206,4 +211,28 @@ const PathwayVisualizer: FC = () => {
   );
 };
 
+const McodeVisualizer: FC = () => {
+  const mcode = usePatientRecords().mcodeRecords;
```

Comment:
We should be consistent with how we use specific parts of the context. I have seen this as well as 
`const {element} = useProvider()`